<!DOCTYPE html>
<html lang="en"
>
<head>
    <title><span class="caps">C.V.</span> - Jacob Frelinger's Blog</title>
    <!-- Using the latest rendering mode for IE -->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">



<link rel="canonical" href="http://jfrelinger.github.io/pages/cv.html">

        <meta name="author" content="Jacob Frelinger" />
        <meta name="description" content="Curriculum Vitae of Jacob Frelinger" />

    <!-- Enable latex plugin -->


    <meta property="og:site_name" content="Jacob Frelinger's Blog" />
    <meta property="og:type" content="article"/>
    <meta property="og:title" content="C.V."/>
    <meta property="og:url" content="http://jfrelinger.github.io/pages/cv.html"/>
    <meta property="og:description" content="Curriculum Vitae of Jacob Frelinger" />


    <!-- Bootstrap -->
        <link rel="stylesheet" href="http://jfrelinger.github.io/theme/css/bootstrap.flatly.min.css" type="text/css"/>
    <link href="http://jfrelinger.github.io/theme/css/font-awesome.min.css" rel="stylesheet">

    <link href="http://jfrelinger.github.io/theme/css/pygments/solarizedlight.css" rel="stylesheet">
        <link href="http://jfrelinger.github.io/theme/css/typogrify.css" rel="stylesheet">
    <link rel="stylesheet" href="http://jfrelinger.github.io/theme/css/style.css" type="text/css"/>

        <link href="http://jfrelinger.github.io/feeds/all.atom.xml" type="application/atom+xml" rel="alternate"
              title="Jacob Frelinger's Blog ATOM Feed"/>



</head>
<body>

<div class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a href="http://jfrelinger.github.io/" class="navbar-brand">
Jacob Frelinger's Blog            </a>
        </div>
        <div class="collapse navbar-collapse navbar-ex1-collapse">
            <ul class="nav navbar-nav">
                         <li><a href="http://jfrelinger.github.io/pages/about-jacob.html">
                             About&nbsp;Jacob
                          </a></li>
                         <li class="active"><a href="http://jfrelinger.github.io/pages/cv.html">
                             <span class="caps">C.V.</span>
                          </a></li>
                        <li >
                            <a href="http://jfrelinger.github.io/category/articles.html">Articles</a>
                        </li>
            </ul>
            <ul class="nav navbar-nav navbar-right">
                <li><a href="http://jfrelinger.github.io/archives.html"><i class="fa fa-th-list"></i><span class="icon-label">Archives</span></a></li>
            </ul>
        </div>
        <!-- /.navbar-collapse -->
    </div>
</div> <!-- /.navbar -->

<div class="container">
    <div class="row">
        <div class="col-sm-9">

    <section id="content" class="body">
        <h1 class="entry-title"><span class="caps">C.V.</span></h1>
        
        <div class="entry-content">
            <div class="section" id="jacob-frelinger">
<h2>Jacob&nbsp;Frelinger</h2>
<table border="1" class="docutils">
<colgroup>
<col width="33%" />
<col width="27%" />
<col width="40%" />
</colgroup>
<tbody valign="top">
<tr><td><p class="first"><a class="reference external" href="mailto:jacob.frelinger&#64;gmail.com">jacob.frelinger&#64;gmail.com</a></p>
<p class="last"><a class="reference external" href="http://jfrelinger.github.io">http://jfrelinger.github.io</a></p>
</td>
<td><p class="first">Contact info&nbsp;available</p>
<p class="last">upon&nbsp;request.</p>
</td>
<td><img alt="Head shot" class="first last align-right" src="../images/head_shot.jpg" style="width: 20%;" />
</td>
</tr>
</tbody>
</table>
<div class="section" id="objective">
<h3>Objective</h3>
<p>Full-time employment working on software for scientific&nbsp;applications</p>
</div>
<div class="section" id="education">
<h3>Education</h3>
<table class="docutils field-list" frame="void" rules="none">
<col class="field-name" />
<col class="field-body" />
<tbody valign="top">
<tr class="field"><th class="field-name">2008-20013:</th><td class="field-body">PhD in Computational Biology and Bioinformatics at <a class="reference external" href="http://www.duke.edu">Duke University</a></td>
</tr>
<tr class="field"><th class="field-name">2004-2009:</th><td class="field-body">BSc in Applied Mathematics at <a class="reference external" href="http://www.ncsu.edu">North Carolina State University</a></td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="technical-skills">
<h3>Technical&nbsp;skills</h3>
<table class="docutils field-list" frame="void" rules="none">
<col class="field-name" />
<col class="field-body" />
<tbody valign="top">
<tr class="field"><th class="field-name" colspan="2">Operating Systems:</th></tr>
<tr class="field"><td>&nbsp;</td><td class="field-body">Unix/Linux, Windows, <span class="caps">OS</span> X</td>
</tr>
<tr class="field"><th class="field-name" colspan="2">Programing Languages:</th></tr>
<tr class="field"><td>&nbsp;</td><td class="field-body">Strong knowledge of C, and Python, good knowledge of
javascript, R, C++, Shell Scripting</td>
</tr>
<tr class="field"><th class="field-name" colspan="2">Project Management:</th></tr>
<tr class="field"><td>&nbsp;</td><td class="field-body">Mercurial, Git, <span class="caps">CVS</span>, <span class="caps">SVN</span></td>
</tr>
<tr class="field"><th class="field-name">DataBase:</th><td class="field-body"><span class="caps">SQL</span>, sqlite, Posgres, MySQL</td>
</tr>
<tr class="field"><th class="field-name">Software:</th><td class="field-body">Office tools, LaTeX</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="experience">
<h3>Experience</h3>
<p><strong>Post-doctoral Research Associate</strong>, <em>September 2013 - Present, Seattle <span class="caps">WA</span></em></p>
<p><a class="reference external" href="http://www.fredhutch.org">Fred Hutchinson Cancer Research&nbsp;Center</a></p>
<ul class="simple">
<li>Develop novel methods for automating analysis of flow cytometry&nbsp;data</li>
</ul>
<p><strong>Graduate Research Assistant</strong>, <em>July 2005 - September 2013, Durham <span class="caps">NC</span></em></p>
<p><a class="reference external" href="http://www.duke.edu">Duke&nbsp;University</a></p>
<ul class="simple">
<li>Develop novel method for automated analysis of flow cytometry&nbsp;data</li>
<li>Develop software for handling and analyzing flow cytometry&nbsp;data</li>
</ul>
<p><strong>Bioinformatics research assistant</strong>, <em>May 2004 - March 2006, Raleigh <span class="caps">NC</span></em></p>
<p><a class="reference external" href="http://www.ncsu.edu">North Carolina State&nbsp;University</a></p>
<ul class="simple">
<li>Assist in development of bioinformatics databases and software for sequencing&nbsp;genomes</li>
<li>Development of web accessible database of GenBank&nbsp;records</li>
</ul>
<p><strong>Tech Geek</strong>, <em>October 2001 - July 2003, Raleigh <span class="caps">NC</span></em></p>
<p>PotsTek</p>
<ul class="simple">
<li>Embedded Linux programmer and system&nbsp;administrator.</li>
</ul>
<p><strong>Systems Administrator</strong>, <em>April 2001 -July 2001, Raleigh <span class="caps">NC</span></em></p>
<p><span class="caps">BTI</span></p>
<ul class="simple">
<li>System administration, computer security and support tool&nbsp;development</li>
</ul>
<p><strong>Systems Adminiatrator</strong>, <em>February 2000 - December, Raleigh <span class="caps">NC</span></em></p>
<p>NewSouth&nbsp;Communications</p>
<ul class="simple">
<li>System administration and database web application&nbsp;development</li>
</ul>
<p><strong>Junior Systems Administrator</strong>, <em>1998 - 1999, Raleigh <span class="caps">NC</span></em></p>
<p>Interpath&nbsp;Communications</p>
<ul class="simple">
<li>System administration and development of support&nbsp;tools</li>
</ul>
</div>
<div class="section" id="publications">
<h3>Publications</h3>
<p><a class="reference external" href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003806"><strong>OpenCyto: An Open Source Infrastructure for Scalable, Robust, Reproducible, and Automated, End-to-End Flow Cytometry Data&nbsp;Analysis</strong></a></p>
<p><em>PLoS Computational Biology</em>
2014
Finak G., <strong>Frelinger J.</strong>, Jaing W., Newell E., Ramey J., Davis M., Kalams S., De Rosa S., Gottardo&nbsp;R.</p>
<p><a class="reference external" href="http://www.sciencedirect.com/science/article/pii/S0022175914001185"><strong>Setting objective thresholds for rare event detection in flow&nbsp;cytometry.</strong></a></p>
<p><em>Journal of immunological methods</em>&nbsp;2014</p>
<p>Richards J., Staas J., Enzor J., McKinnon K., <strong>Frelinger J.</strong>, Denny&nbsp;T.</p>
<p><a class="reference external" href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003130"><strong>Hierarchical Modeling for Rare Event Detection and Cell Subset Alignment across Flow Cytometry&nbsp;Samples</strong></a></p>
<p><em><span class="caps">PLOS</span> Computational Biology</em>&nbsp;2013</p>
<p>Cron A., Gouttefangeas C., <strong>Frelinger J.</strong>, Lin L., Singh S., Britten C., Welters M., van der Burg S., West M., Chan&nbsp;C.</p>
<p><a class="reference external" href="http://amstat.tandfonline.com/doi/abs/10.1198/jcgs.2010.10016#.VCtoRHTn8hc"><strong>Understanding <span class="caps">GPU</span> programming for statistical computation: Studies in massively parallel massive&nbsp;mixtures</strong></a></p>
<p><em>Journal of Computational and Graphical Statistics</em>&nbsp;2010</p>
<p>Suchard M., Wang Q., Chan C., <strong>Frelinger J.</strong>, Cron A., West&nbsp;M.</p>
<!-- This article describes advances in statistical computation for large-scale data analysis in structured Bayesian mixture models via graphics processing unit (GPU) programming. The developments are partly motivated by computational challenges arising in fitting models of increasing heterogeneity to increasingly large datasets. An example context concerns common biological studies using high-throughput technologies generating many, very large datasets and requiring increasingly high-dimensional mixture models with large numbers of mixture components. We outline important strategies and processes for GPU computation in Bayesian simulation and optimization approaches, give examples of the benefits of GPU implementations in terms of processing speed and scale-up in ability to analyze large datasets, and provide a detailed, tutorial-style exposition that will benefit readers interested in developing GPU-based approaches in other statistical models. Novel, GPU-oriented approaches to modifying existing algorithms software design can lead to vast speed-up and, critically, enable statistical analyses that presently will not be performed due to compute time limitations in traditional computational environments. Supplemental materials are provided with all source code, example data, and details that will enable readers to implement and explore the GPU approach in this mixture modeling context. -->
<p><a class="reference external" href="http://onlinelibrary.wiley.com/doi/10.1002/cyto.a.20987/full"><strong>Optimization of a highly standardized carboxyfluorescein succinimidyl ester flow cytometry panel and gating strategy design using discriminative information measure&nbsp;evaluation</strong></a></p>
<p><em>Cytometry Part A</em>&nbsp;2010</p>
<p>Chan C., Lin L., <strong>Frelinger J.</strong>, Hérber V., Gagnon D., Landry C., Sékal R., Enzor J., Staats J., Weinhold K., Jaimes M., West&nbsp;M.</p>
<!-- The design of a panel to identify target cell subsets in flow cytometry can be difficult when specific markers unique to each cell subset do not exist, and a combination of parameters must be used to identify target cells of interest and exclude irrelevant events. Thus, the ability to objectively measure the contribution of a parameter or group of parameters toward target cell identification independent of any gating strategy could be very helpful for both panel design and gating strategy design. In this article, we propose a discriminative information measure evaluation (DIME) based on statistical mixture modeling; DIME is a numerical measure of the contribution of different parameters towards discriminating a target cell subset from all the others derived from the fitted posterior distribution of a Gaussian mixture model. Informally, DIME measures the "usefulness" of each parameter for identifying a target cell subset. We show how DIME provides an objective basis for inclusion or exclusion of specific parameters in a panel, and how ranked sets of such parameters can be used to optimize gating strategies. An illustrative example of the application of DIME to streamline the gating strategy for a highly standardized carboxyfluorescein succinimidyl ester (CFSE) assay is described -->
<p><a class="reference external" href="http://link.springer.com/article/10.1007%2Fs00262-010-0883-4"><strong>Modeling flow cytometry data for cancer vaccine immune&nbsp;monitoring</strong></a></p>
<p><em>Cancer Immunology, Immunotherapy</em>&nbsp;2010</p>
<p><strong>Frelinger J.</strong>, Ottinger J., Gouttefangeas C, Chan&nbsp;C.</p>
<!-- Flow cytometry (FCM) is widely used in cancer research for diagnosis, detection of minimal residual disease, as well as immune monitoring and profiling following immunotherapy. In all these applications, the challenge is to detect extremely rare cell subsets while avoiding spurious positive events. To achieve this objective, it helps to be able to analyze FCM data using multiple markers simultaneously, since the additional information provided often helps to minimize the number of false positive and false negative events, hence increasing both sensitivity and specificity. However, with manual gating, at most two markers can be examined in a single dot plot, and a sequential strategy is often used. As the sequential strategy discards events that fall outside preceding gates at each stage, the effectiveness of the strategy is difficult to evaluate without laborious and painstaking back-gating. Model-based analysis is a promising computational technique that works using information from all marker dimensions simultaneously, and offers an alternative approach to flow analysis that can usefully complement manual gating in the design of optimal gating strategies. Results from model-based analysis will be illustrated with examples from FCM assays commonly used in cancer immunotherapy laboratories. -->
<p><a class="reference external" href="http://link.springer.com/article/10.1186%2F1751-0473-3-10"><strong>Flow: Statistics, visualization and informatics for flow&nbsp;cytometry</strong></a></p>
<p><em>Source Code for Biology and&nbsp;Medicine</em></p>
<p>2008</p>
<p><strong>Frelinger J.</strong>, Kepler T., Chan&nbsp;C.</p>
<!-- Flow is an open source software application for clinical and experimental researchers to perform exploratory data analysis, clustering and annotation of flow cytometric data. Flow is an extensible system that offers the ease of use commonly found in commercial flow cytometry software packages and the statistical power of academic packages like the R BioConductor project. -->
</div>
</div>

        </div>
    </section>
        </div>
        <div class="col-sm-3 well well-sm" id="sidebar">

<aside>
    <section>
        <ul class="list-group list-group-flush">
                <li class="list-group-item"><h4><i class="fa fa-home fa-lg"></i><span class="icon-label">Social</span></h4>
                  <ul class="list-group" id="social">
                    <li class="list-group-item"><a href="https://github.com/jfrelinger"><i class="fa fa-github-square fa-lg"></i> Github</a></li>
                    <li class="list-group-item"><a href="https://www.linkedin.com/in/jfrelinger"><i class="fa fa-linkedin-square fa-lg"></i> LinkedIn</a></li>
                  </ul>
                </li>



                <li class="list-group-item"><a href="http://jfrelinger.github.io/"><h4><i class="fa fa-tags fa-lg"></i><span class="icon-label">Tags</span></h4></a>
                    <ul class="list-group " id="tags">
                        <li class="list-group-item tag-4">
                            <a href="http://jfrelinger.github.io/tag/r.html">
                                R
                            </a>
                        </li>
                        <li class="list-group-item tag-4">
                            <a href="http://jfrelinger.github.io/tag/python.html">
                                python
                            </a>
                        </li>
                        <li class="list-group-item tag-4">
                            <a href="http://jfrelinger.github.io/tag/prime-numbers.html">
                                prime numbers
                            </a>
                        </li>
                        <li class="list-group-item tag-4">
                            <a href="http://jfrelinger.github.io/tag/opencyto.html">
                                OpenCyto
                            </a>
                        </li>
                        <li class="list-group-item tag-4">
                            <a href="http://jfrelinger.github.io/tag/iterators.html">
                                iterators
                            </a>
                        </li>
                    </ul>
                </li>    
    <li class="list-group-item"><h4><i class="fa fa-external-link-square fa-lg"></i><span class="icon-label">Links</span></h4>
      <ul class="list-group" id="links">
        <li class="list-group-item">
            <a href="http://python.org/" target="_blank">
                Python.org
            </a>
        </li>
        <li class="list-group-item">
            <a href="https://code.google.com/p/py-fcm/" target="_blank">
                fcm
            </a>
        </li>
        <li class="list-group-item">
            <a href="http://opencyto.org" target="_blank">
                OpenCyto
            </a>
        </li>
        <li class="list-group-item">
            <a href="http://getpelican.com/" target="_blank">
                Pelican
            </a>
        </li>
      </ul>
    </li>

        </ul>
    </section>

</aside>        </div>
    </div>
</div>
<footer>
   <div class="container">
      <hr>
      <div class="row">
         <div class="col-xs-10">&copy; 2014 Jacob Frelinger
            &middot; Powered by <a href="https://github.com/DandyDev/pelican-bootstrap3" target="_blank">pelican-bootstrap3</a>,
            <a href="http://docs.getpelican.com/" target="_blank">Pelican</a>,
            <a href="http://getbootstrap.com" target="_blank">Bootstrap</a>         </div>
         <div class="col-xs-2"><p class="pull-right"><i class="fa fa-arrow-up"></i> <a href="#">Back to top</a></p></div>
      </div>
   </div>
</footer>
<script src="http://jfrelinger.github.io/theme/js/jquery.min.js"></script>

<!-- Include all compiled plugins (below), or include individual files as needed -->
<script src="http://jfrelinger.github.io/theme/js/bootstrap.min.js"></script>

<!-- Enable responsive features in IE8 with Respond.js (https://github.com/scottjehl/Respond) -->
<script src="http://jfrelinger.github.io/theme/js/respond.min.js"></script>

</body>
</html>